backend/server.py

```diff
@@ -113,15 +115,26 @@ def compute_running_avg(trace, k=100):
     return avgs
     
 
-def compute_stdev(trace):
+def compute_stdev(trace, k=100):
```

Comment:
Document k. Is there a better name that could be used? i/k/n starts to get confusing.